ujt/config.py

```diff
@@ -0,0 +1,31 @@
+import configparser
+
+# default values
+CLEAR_CACHE_ON_STARTUP = True
+REFRESH_TOPOLOGY_ON_STARTUP = True
+
+AUTO_REFRESH_SLI = True
+CLIENT_SLI_REFRESH_INTERVAL_MILLIS = 30 * 1000  # ms
+
+REPORTING_SERVER_ADDRESS = "localhost:50051"
+
+
+def load_config(config_path):
+    if config_path is None:
+        return
+
+    config = configparser.ConfigParser()
+    config.read(config_path)
+
+    global CLEAR_CACHE_ON_STARTUP, REFRESH_TOPOLOGY_ON_STARTUP, AUTO_REFRESH_SLI, CLIENT_SLI_REFRESH_INTERVAL_MILLIS, REPORTING_SERVER_ADDRESS
```

Comment:
I asked John about this and he said it seems fine to expose options via global vars, since [flags](https://abseil.io/docs/python/guides/flags) does something similar and is used in google.